editor/src/clj/editor/pipeline/tex_gen.clj

```diff
@@ -90,12 +91,11 @@
                    images)))
 
 (defn assemble-texture-images
-  ^Graphics$TextureImage [texture-images max-page-count]
-  (let [texture-images (into-array Graphics$TextureImage texture-images)
-        texture-type (if (pos? max-page-count)
+  ^TextureGenerator$GenerateResult [texture-generator-results max-page-count]
+  (let [texture-type (if (pos? max-page-count)
                        Graphics$TextureImage$Type/TYPE_2D_ARRAY
                        Graphics$TextureImage$Type/TYPE_2D)]
-    (TextureUtil/createCombinedTextureImage texture-images texture-type)))
+    (TextureUtil/createCombinedTextureImage (into-array texture-generator-results) texture-type)))
```

Comment:
`TextureUtil/createCombinedTextureImage` is called in the `assemble-cubemap-texture-images` function below as well, so you need to update that to return a `TextureGenerator$GenerateResult` and all callers of that too.